hotness/consumers.py

```diff
@@ -221,21 +219,19 @@ def _handle_anitya_update(self, upstream, package, msg):
         is_monitored = self.is_monitored(package)
 
         # Is it new to us?
-        fname = self.yumconfig
-        try:
-            version, release = hotness.repository.get_version(
-                package, fname, self.package_manager)
-        except KeyError:
-            # At this point, we have tried very hard to find the rawhide
-            # version of the package.  If we didn't find it, that means there
-            # likely hasn't yet been a rawhide build of it.. and there's
-            # nothing reasonable we can do.  Notify the world of our failure
-            # and go back to the event loop.
-            self.log.warn("No rawhide version found for %r" % package)
+        mdapi_url = '{0}/srcpkg/koji/{1}'.format(self.mdapi_url, package)
+        self.log.debug("Getting pkg info from %r" % mdapi_url)
+        r = requests.get(mdapi_url)
+        if r.status != 200:
+            # Unfortunately it's not in mdapi, we can't do much about it
+            self.log.warning("No koji version found for %r" % package)
```

Comment:
I don't think so. If package not in koji repo, then I don't see point why we should touch rawhide. I guess this is the case for retired packages.